### PR TITLE
INT-308 | Fix styling for hero slider links

### PR DESCRIPTION
**Pull request by perjg**
Updated styling for links in the hero slider to match design document.

### CODE REVIEW

File: homepage/front/styles/component/_hero.scss [hunk 1]
```diff
@@ -264,3 +265,21 @@ $desktop-slider: new-breakpoint(min-width 1001px);
...
+	background: url('/images/arrow-right.png') no-repeat 0 0;
+	display: inline-block;
+	height: 16px;
+	width: 16px;
+}
+
+.hero-link {
+	color: white;
+	cursor: pointer;
+	font-size: 1.3em;
```
Comment (kenkouot) [inline homepage/front/styles/component/_hero.scss]: We use rems for our font sizes. Easier to reason about.

File: homepage/front/styles/component/_hero.scss [hunk 2]
```diff
@@ -264,3 +265,21 @@ $desktop-slider: new-breakpoint(min-width 1001px);
...
+	margin-top: 1em;
```
Comment (kenkouot) [inline homepage/front/styles/component/_hero.scss]: Just use px for box positioning.

**Comment by perjg** [comment]
@kenkouot I've changed the font size to rem, and I removed padding-top as it wasn't needed for the layout.

**Comment by perjg** [comment]
@kenkouot I needed these changes to implement INT-310, so I included the commits here in the PR for that issue: https://github.com/Wikia/mercury/pull/1881

I'll close down this one and merge everything together in INT-310.

**Comment by perjg** [comment]
Will be merged together with INT-310.

**Pull request closed without merging**